docs/pages/blog/mui-x-v8.md

```diff
@@ -0,0 +1,457 @@
+</figure>
+
+Learn how to add [Lazy loading](/x/react-tree-view/rich-tree-view/lazy-loading/) to your application's Tree View.
+
+### Performance improvements in re-rendering
+
+We've optimized the Tree View to update only the parts that need changes, rather than re-rendering all items.
+With the use of selectors to precisely target nodes for re-rendering, we've achieved a significant performance boost.
+In upcoming releases, we'll make another big step with the introduction of virtualization.
+
+### Automatic selection propagation
+
+The Rich Tree View now supports parent/child selection propagation.
+This means you can select a parent and automatically select all of its children, and vice versa—depending on your requirements and configuration.
+
+<figure>
+  <video autoplay muted loop playsinline style="width: 300px" width="704" height="720">
+    <source alt="Automatic selection of children"  src="/static/blog/mui-x-v8/tree-view-automatic-selection.mp4" type="video/mp4">
+  </video>
+  <figcaption>Automatic selection of children</figcaption>
+</figure>
+
+```tsx
+type TreeViewSelectionPropagation = {
+  descendants?: boolean; // default: false
+  parents?: boolean; // default: false
+};
+```
+
+You can learn more in the [Selection docs](/x/react-tree-view/rich-tree-view/selection/).
+
+### New customization hook
+
+We've officially transitioned to a new developer experience for customizing Tree Items for the Rich Tree View.
+With our new customization hook, you can freely compose your Tree Item while still leveraging all the out-of-the-box features of the Rich Tree View.
+This provides unparalleled flexibility to tailor your tree components to your project's unique requirements.
+
+```js
+const CustomTreeItemComponent = React.forwardRef(function CustomTreeItemComponent(
+  { id, itemId, label, disabled, children }: TreeItemProps,
+  ref: React.Ref<HTMLLIElement>,
+) {
+  const treeItemData = useTreeItem({ id, itemId, children, label, disabled, rootRef: ref });
+
+  return (
+    <TreeItemProvider {...treeItemData.getContextProviderProps()}>
+      <TreeItemRoot {...treeItemData.getRootProps()}>
+        <TreeItemContent {...treeItemData.getContentProps()}>
+          <TreeItemLabel {...treeItemData.getLabelProps()} />
+        </TreeItemContent>
+        {children && <TreeItemGroupTransition {...treeItemData.getGroupTransitionProps()} />}
+      </TreeItemRoot>
+    </TreeItemProvider>
+  );
+}
+```
+
+Learn more this and other options on [Tree Item Customization docs](/x/react-tree-view/tree-item-customization/).
+
+## Charts
+
+### Funnel Charts [<span class="plan-pro"></span>](/x/introduction/licensing/#pro-plan 'Pro plan')
+
+We have expanded our chart portfolio with the addition of Funnel Charts, providing a new way to visualize data conversions and drop-offs.
+
+<figure>
+  <img alt="Funnel charts examples" src="/static/blog/mui-x-v8/funnel-charts.png" loading="lazy" width="1880" height="521">
```

Comment:
It might be a bit miss leading to display figma screenshots that are not implemented, and not sure they will be available

Reply:
why wouldn't they be available?

Reply:
Or maybe the question is, when would they be available?

Reply:
The one with the rectangle does not match the current implementation. It would be more appropriate to have a full rendering based on HTML instead of SVG.

For the pyramid, the question is "when would it be available"

Reply:
I'm replacing it due to the uncertainty of when,
But the current design is below standard and we should fix at least the color value of the font related to the pallete ASAP.